api/types/types.go

```diff
@@ -45,6 +45,12 @@ type ImageInspect struct {
 	VirtualSize     int64
 	GraphDriver     GraphDriverData
 	RootFS          RootFS
+	Metadata        ImageMetadata
+}
+
+// ImageMetadata contains engine-local data about the image
+type ImageMetadata struct {
+	LastTagTime string
```

Comment:
We seem to use string sometimes and `time.Time` other times. I don't think `time.Time` is correct, because it doesn't default to the `time.RFC3339Nano` format.